src/utils/gwhisperUtils.cpp

```diff
@@ -0,0 +1,31 @@
+#include "gwhisperUtils.hpp"
+#include <fstream>
+namespace gwhisper
+{
+    namespace util
+    {
+
+        /// Utility-Function for reading contents of a file and returning them as a string
```

Comment:
document function in header file only, otherwise documentation is hard to keep in sync